web/text_layer_builder.js

```diff
@@ -114,6 +119,7 @@ class TextLayerBuilder {
         this.textLayerDiv.append(textLayerFrag);
         this._finishRendering();
         this.highlighter?.enable();
+        this.accessibilityManager?.onTextLayerRendered();
```

Comment:
Generally speaking, let's make the `AccessibilityManager` a bit more similar to the `TextHighlighter` class.
Hence, let's call this method `enable` instead. (You might also want to introduce a `disable` method, to handle any clean-up when the textLayer goes away.)

Reply:
Another thing, that could perhaps simplify things and reduce the number of DOM lookups in the `AccessibilityManager`-code, would be to again follow the `TextHighlighter`-class and pass in the textLayer-divs similar to: https://github.com/mozilla/pdf.js/blob/4f6cd05a53e6c069bb4ebee469fda6437a0971af/web/text_layer_builder.js#L99